src/main/resources/australia.json

```diff
@@ -0,0 +1,116 @@
+[
+    {
+        "name": "Australian National University",
+        "country": "Australia",
+        "address": "Canberra ACT 2601, Australia",

```

Comment:
For the other countries we do not include the country name in the address. Let's remove for consistency.